components/file_system/src/lib.rs

```diff
@@ -130,6 +132,13 @@ impl std::ops::Sub for IoBytes {
     }
 }
 
+impl std::ops::AddAssign for IoBytes {
+    fn add_assign(&mut self, rhs: Self) {
+        self.read += rhs.read;
```

Comment:
maybe use aturating_add

Reply:
Because the final result is to use the sub delta, overflow add is better here. But anyway, this addassign is only use for test convenient, so the difference is not much.